lib/between_meals/repo/hg.rb

```diff
@@ -167,37 +167,37 @@ def parse_status(changes)
 
         changes.lines.map do |line|
```

Comment:
if we're going to do this rather than regexing it like that, we can should do:
```ruby
parts = line.split("\t")
case parts[0]
when 'A'
  {
    :status => :added
    :path => parts[1]
  }
  ...
```

In addition to not regexing things, it also means that if there's a 3rd or 4th field then we still parsed the filename correctly.